webapp/src/components/common/Matrix/components/MatrixResizer/index.tsx

```diff
@@ -0,0 +1,98 @@
+/**
+ * Copyright (c) 2025, RTE (https://www.rte-france.com)
+ *
+ * See AUTHORS.txt
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ *
+ * SPDX-License-Identifier: MPL-2.0
+ *
+ * This file is part of the Antares project.
+ */
+
+import { useState } from "react";
+import { Box, TextField } from "@mui/material";
+import LoadingButton from "@mui/lab/LoadingButton";
+import { replaceMatrix } from "@/services/api/matrix";
+import type { NonEmptyMatrix } from "../../shared/types";
+import { useUpdateEffect } from "react-use";
+import Transform from "@mui/icons-material/Transform";
+import { resizeMatrix } from "../../shared/utils";
+import { useTranslation } from "react-i18next";
+import useEnqueueErrorSnackbar from "@/hooks/useEnqueueErrorSnackbar";
+import { toError } from "@/utils/fnUtils";
+
+interface MatrixResizerProps {
+  studyId: string;
+  path: string;
+  data: NonEmptyMatrix;
+  onMatrixUpdated: VoidFunction;
+}
+
+function MatrixResizer({ studyId, path, data, onMatrixUpdated }: MatrixResizerProps) {
```

Comment:
It's an action, better to move it and use it in MatrixActions

Reply:
It will fix the height issue, and allows to move it after the save

![image](https://github.com/user-attachments/assets/2834061c-5965-427a-bfcc-ecd198deb093)